### ISSUE TITLE
`process_tasks` fails when `DEBUG=True`

### ISSUE THREAD

**Issue by chadgh** [arteria/django-background-tasks#12]
I get the following error and traceback when I try to run `./manage.py process_tasks` when DEBUG=True.

```
Traceback (most recent call last):
  File "./manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/home/chadgh/.virtualenvs/atom/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/chadgh/.virtualenvs/atom/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/chadgh/.virtualenvs/atom/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/chadgh/.virtualenvs/atom/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/home/chadgh/.virtualenvs/atom/lib/python2.7/site-packages/background_task/management/commands/process_tasks.py", line 76, in handle
    autodiscover()
  File "/home/chadgh/.virtualenvs/atom/lib/python2.7/site-packages/background_task/tasks.py", line 234, in autodiscover
    app_path = import_module(app).__path__
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named DebugToolbarConfig
```

**Comment by philippeowagner** [comment]
Thanks @chadgh for reporting this issue. 
Could you please post your freezed requirement file and what's in your ``settings.py`` related to [django-debug-toolbar](https://github.com/django-debug-toolbar/django-debug-toolbar)?

**Comment by chadgh** [comment]
I think I got it figured out somewhat.

I had in my INSTALLED_APPS:
`'debug_toolbar.apps.DebugToolbarConfig'`

I changed that to `'debug_toolbar'` and the command doesn't fail now.

Things still don't seem to be working though. If I run the command `./manage.py process_tasks --verbosity=3` other than a few deprication warnings I don't see any output. My tasks indicate that they have had several attempts and as the times comes and then passes the run_at value gets bumped up, the attempts values gets bumped up and I get no output from the management command.

Should I create a new ticket for this?

**Comment by philippeowagner** [comment]
Great that it works now using ``debug_toolbar`` in the ``INSTALLED_APPS``. I was a bit surprised because we use 'django-background-tasks' in combination with 'debug_toolbar'. 

We released a bugfix (version 1.0.2) yesterday that fixes an error related to timezones. The error occurred in ``run_at``. Could you please re-try using the latest version and create a new issue in case of problems. Thanks again for providing feedback related to 'django-background-tasks'.
